packages/chips/Chip.tsx

```diff
@@ -23,7 +23,7 @@ import * as React from 'react';
 import * as classnames from 'classnames';
 import * as Ripple from '@material/react-ripple';
 // @ts-ignore no mdc .d.ts file
-import {MDCChipFoundation} from '@material/chips/dist/mdc.chips';
+import {MDCChipFoundation} from '@material/chips/chip/foundation';
```

Comment:
I'm not sure if we can make these changes. `@material/chips/chip/foundation` is all ES6 code. People targeting a browser that only supports ES5 might not be able to use this package anymore, since the build step does not transpile mdc web's chip/foundation.  

I think we'll need to wait until MDC Web typescript conversion is complete.

Reply:
This pull request bundles @material/ dependencies into react material. It this case it will be transpiled to ES5